### PR TITLE
Create araza-github.json

### PR DESCRIPTION
**Pull request by araza-github**

### CODE REVIEW

File: data/araza-github.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "bio": "Salesforce Developer | Salesforce Certified Administrator | Social Media",
+  "links": [
+      {
+        "name": "Twitter",
+        "url": "https://twitter.com/araza_malik",
+        "icon": "FaTwitter"
+      },{
+        "name": "LinkedIn",
+        "url": "https://twitter.com/araza_malik",
+        "icon": "FaLinkedIn"
```
Comment (imbeshat) [inline data/araza-github.json:13]:
```suggestion
        "icon": "FaLinkedin"
```

**Review by imbeshat — CHANGES_REQUESTED** [review]

File: data/araza-github.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "type": "personal",
+  "bio": "Salesforce Developer | Salesforce Certified Administrator | Social Media",
+  "links": [
+      {
+        "name": "Twitter",
+        "url": "https://twitter.com/araza_malik",
+        "icon": "FaTwitter"
+      },{
+        "name": "LinkedIn",
+        "url": "https://twitter.com/araza_malik",
```
Comment (vedprakashnautiyal) [inline data/araza-github.json:12]:
```suggestion

```

Comment (vedprakashnautiyal) [reply data/araza-github.json:12]: You should add linkedin link here

**Review by vedprakashnautiyal — CHANGES_REQUESTED** [review]
Please review your links and icons

File: data/araza-github.json (hunk 2 above)
Comment (araza-github) [reply data/araza-github.json:12]: okay Thank you

**Pull request merged**